internal/authentication/tokengetter_test.go

```diff
@@ -85,3 +85,65 @@ func TestTokenGetterGet(t *testing.T) {
 		}
 	}
 }
+
+func BenchmarkTokenGetterGet(b *testing.B) {
+	fakeClient := fake.NewSimpleClientset()
+	fakeClient.PrependReactor("create", "serviceaccounts/token",
+		func(action ctest.Action) (bool, runtime.Object, error) {
+			act, ok := action.(ctest.CreateActionImpl)
```

Comment:
It might make sense to stop the benchmark timer for the duration of this function. Otherwise, this code will be included in the benchmark results.

Reply:
Removing this test and implementing benchmarks for more consequential functions